scripts/5.meta.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+if [ -z $BASE_DIR ]; then
+	echo "Please run 'source .env', and confirm you are running the script inside Docker."
+	exit
+fi
+
+WORKING_DIR="$BASE_DIR/temp/demo_output"
+MODULE="meta"
+
+ELEMENTS_IN="[\"local\"]"
+
+CONFIG=$( jq -n \
+	--argjson el_in "$ELEMENTS_IN" \
+	'{
+		elements_in: $el_in,
+		delete_cache : true,
+		modules: [
```

Comment:
maybe make this `analysers`, or `children`.

Reply:
Changed to `children` to be consistent with naming in meta analyser code